kong/plugins/zipkin/schema.lua

```diff
@@ -11,6 +11,7 @@ return {
 					{ sample_ratio = { type = "number",
 					                   default = 0.001,
 					                   between = { 0, 1 } } },
+					{ include_credential = typedefs.boolean { default = true } },
```

Comment:
There's no typedef for boolean.
```suggestion
					{ include_credential = { type = "boolean", required = true, default = true } },
```

Reply:
ahhh, it's probably not needed for _record_ fields (for top-level fields when one adds a default the column would contain `NULL`). Still, it doesn't hurt to have. I don't recall of the top of my head, but I think this may prevent the weird `include_credential = ngx.null`.

Reply:
> but I think this may prevent the weird `include_credential = ngx.null`.

I'm not sure if that's something that needs to be prevented? To me that would be "please just use the default, whatever that may mean in future versions".